bin/nit-makepackageini

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script creates (or override) a package.ini template file in the current directory.
+#
+# Default values are guessed from git and the file system.
+
+name=`basename $PWD`
+maint=`git shortlog -esn . | head -n 1 | sed 's/\s*[0-9]*\s*//'`
+dir=`git rev-parse --show-prefix`
+remote=`git remote get-url origin`
+root=${remote%.git}
+
+cat > package.ini <<FIN
```

Comment:
* Please use an english identifier (like `EOF`) or apostrophes.
* Why not `printf` or `echo`?